src/gateway/kong-manager/auth/ldap/ldap-service-directory-mapping.md

```diff
@@ -0,0 +1,154 @@
+
+## Group-role assignment
+
+Using service directory mapping, groups are mapped to roles. When a user logs in, they are identified with their admin username and authenticated with the matching user credentials in the service directory.
+The groups in the service directory are then automatically matched to the associated roles that the organization has defined.
+
+### Example
+
+1. Example Corp maps the service directory group, `T1-Mgmt`, to the Kong role super-admin.
+2. Example Corp maps a service directory user, named `example-user`, to a Kong admin account with the same name, `example-user`.
+3. The user, `example-user`, is assigned to the group `T1-Mgmt` in the LDAP Directory.
+
+
+When `example-user` logs in as an admin to Kong Manager, they will automatically have the role of super-admin as a result of the mapping.
+
+If Example Corp decides to revoke `example-user`'s privileges by removing their assignment to `T1-Mgmt`, they will no longer have the super-admin role when they attempt to log in.
+
+## Set up a directory user as the first super admin
+
+Setting up a directory user as the first super admin is recommended by Kong.
+
+The example shows an attribute configured with a unique identifier (UID), and the directory user you want to make the super admin has a distinguished name (DN) entry of `UID=example-user`:
+
+```sh
+curl -i -X PATCH https://localhost:8001/admins/kong_admin \
+  --header 'Kong-Admin-Token: <RBAC_TOKEN>' \
+  --header 'Content-Type: application/json' \
+  --data '{"username":"example-user"}'
+```
+
+This user will be able to log in, but until you map a group belonging to `example-user` to a role, the user will only use the directory for authentication. Once you map the super-admin role to a group that `example-user` is in, then you can delete the super-admin role from the `example-user` admin. The group you pick needs to be “super” in your directory, otherwise as other admins log in with a generic group, for example the “employee” group, they will also become super-admins.
+
+{:.important}
+> **Important**: If you delete the super-admin role from your only admin, and have not yet mapped the super-admin role to a group that admin belongs to, then you will not be able to log in to Kong Manager.
+
+Alternatives:
+
+* Start Kong with RBAC turned off, map a group to the super-admin role, and then create an admin to correspond to a user belonging to that group. Doing so ensures that the super admin's privileges are entirely tied to the directory group, whereas bootstrapping a super admin only uses the directory for authentication.
+
+* Create all admin accounts for matching directory users and ensure that their existing groups map to appropriate roles before enforcing RBAC.
```

Comment:
Do you think this is worth deleting?

Reply:
I honestly don't know. The first point seems potentially useful - something about managing the super admin user through LDAP (you can adjust their roles through the directory) vs boostrapping, which doesn't give you control over the roles. 
The second point seems useless - you can map first and then enforce, or enforce and then map. There's no real difference.